src/ru/nsu/ccfit/zuev/osu/game/HitCircle.java

```diff
@@ -8,6 +8,7 @@
 import org.anddev.andengine.entity.scene.Scene;
 import org.anddev.andengine.entity.sprite.Sprite;
 import org.anddev.andengine.opengl.texture.region.TextureRegion;
+import org.anddev.andengine.util.Debug;
```

Comment:
Perhaps you forgot to remove from testing? 😛

Reply:
Yep